### PR TITLE
Add CoverM tool

### PR DESCRIPTION
**Pull request by mira-miracoli**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [x] - This PR adds a new tool or tool collection
* [ ] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

File: tools/coverm/.shed.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+name: CoverM
```
Comment (gallardoalba) [inline tools/coverm/.shed.yml:1]:
```suggestion
name: coverm
```

File: tools/coverm/.shed.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    CoverM
+    The CoverM tool includes a genome and a contig tool to calculate DNA read coverage and relative abundance focused on metagenomics applications.
+remote_repository_url: https://github.com/galaxyproject/tools-iuc/tools/coverm
+homepage_url: https://github.com/wwood/CoverM
+type: unrestricted
+auto_tool_repositories:
+    name_template: "{{ tool_id }}"
+    description_template: "Wrapper for CoverM tool: {{ tool_name }}."
+suite:
+    name: "CoverM"
```
Comment (gallardoalba) [inline tools/coverm/.shed.yml:16]:
```suggestion
    name: "suite_coverm"
```

File: tools/coverm/coverm_contig.xml [hunk 3]
```diff
@@ -0,0 +1,379 @@
+<tool id="coverm-contig" name="CoverM-CONTIG" version="@TOOL_VERSION@" python_template_version="3.5">
```
Comment (gallardoalba) [inline tools/coverm/coverm_contig.xml:1]:
```suggestion
<tool id="coverm_contig" name="CoverM-CONTIG" version="@TOOL_VERSION@" python_template_version="3.5">
```

File: tools/coverm/coverm_genome.xml [hunk 4]
```diff
@@ -0,0 +1,570 @@
+<tool id="coverm-genome" name="CoverM-GENOME" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" python_template_version="3.5">
```
Comment (gallardoalba) [inline tools/coverm/coverm_genome.xml:1]:
```suggestion
<tool id="coverm_genome" name="CoverM-GENOME" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" python_template_version="3.5">
```

**Review by gallardoalba — COMMENTED** [review]
Congrats, it's a very good job! :D

**Comment by mira-miracoli** [comment]
> Congrats, it's a very good job! :D

Thank you :)

**Review by gallardoalba — APPROVED** [review]

**Pull request merged**